frontend/src/components/Layout/Header/SearchCourses/SearchResult.tsx

```diff
@@ -10,17 +10,26 @@ import { SearchResultWrapper, NoSearchResults } from './styled';
 
 interface CoursesFound {
   coursesFound: Array<Course>;
+  handleSearchClose: () => void;
 }
 
 const LAST_ARRAY_ITEM = -1;
 
-const SearchResult: React.FC<CoursesFound> = ({ coursesFound }) => {
-  const [foundInMyCourses, setFoundInMyCourses] = useState<string | undefined>();
+const SearchResult: React.FC<CoursesFound> = ({ coursesFound, handleSearchClose }) => {
+  const [foundInMyCourses, setFoundInMyCourses] = useState<
+    Array<{ courseId: string; clientCourseId: string }>
+  >([]);
 
   const findCourse = (courses: Array<ClientCourse>, title: string): void => {
```

Comment:
In general, i think that all logic with search courses can be improved. We should discuss it. 

```suggestion
    const findCourse = (courses: ClientCourse[], title: string): void => {
    const foundedCourse = courses.find(({ course }) => course.title === title);

    if (foundedCourse) {
      setFoundInMyCourses((prevState) => [
        ...prevState,
        {
          courseId: foundedCourse.course._id,
          clientCourseId: foundedCourse._id,
        },
      ]);
    }
  };
```